x-pack/plugins/enterprise_search/common/types/engines.ts

```diff
@@ -40,10 +40,24 @@ export interface EnterpriseSearchEngineIndex {
 export interface EnterpriseSearchEngineFieldCapabilities {
   created: string;
   field_capabilities: FieldCapsResponse;
+  fields?: SchemaField[];
   name: string;
   updated: string;
 }
 export interface EnterpriseSearchSchemaField {
   field_name: string;
   field_type: string[];
 }
+
+export interface SchemaFieldIndex {
+  name: string;
+  type: string;
+}
+
+// TODO: This should replace EnterpriseSearchSchemaField once parseFieldsCapabilities is ready
```

Comment:
What needs to be done to clear this TODO?

Reply:
we need to be using the parsed field capabilities in the UI. more of a note than a TODO i think